### DISCUSSION TITLE
Storyboard element negative scaling is inconsistent with stable

### DISCUSSION THREAD

**Discussion by hlysine** [ppy/osu#17227] [Q&A] [answered]
Affected beatmap: https://osu.ppy.sh/beatmapsets/1405100#mania/2897720

<details>
<summary>Videos (See the "playfield borders" at 0:10)</summary>

Stable:

https://user-images.githubusercontent.com/25472513/158053167-25fcf711-15cb-4c02-be89-a6f53ecf3010.mp4

Lazer:

https://user-images.githubusercontent.com/25472513/158053190-b2047c1a-2346-4db5-867f-1f06e3ad75ac.mp4

</details>

Here is an example of a storyboard sprite that causes this bug:
```
Sprite,Overlay,TopLeft,"sb\px.png",240,88
 F,0,49047,,0.1
 C,0,49047,,0,0,0
 V,0,49047,,1,-512
 V,0,55578,,0,-512
```
The difference in behavior between stable and lazer is caused by the negative scale in the `V` command.

Here's a test beatmap to showcase the issue: [Storyboard Debug.osz.zip](https://github.com/ppy/osu/files/8239270/Storyboard.Debug.osz.zip)

**Comment by hlysine** [comment]
There is also another problem noticeable at the start of the videos. Some of the sprites are appearing too early when the playfield changes position. I haven't created a separate report for that because I guess it might be related to #13705. Haven't done any proper debugging to confirm though.

**Comment by frenzibyte** [comment, marked as answer]
Escalated to #17237.
